packages/simple-validators/src/utils.ts

```diff
@@ -1,28 +0,0 @@
-export function getDate(object: undefined | Date | string | { toDate: () => Date }): Date {
-  if (object == null) {
-    return new Date();
-  }
-  if (object instanceof Date) {
-    return object;
-  }
-  if (typeof object === 'string') {
-    return new Date(object);
-  }
-  if (object?.toDate !== undefined) {
-    return object.toDate();
-  }
-  throw new Error(`Could not parse date: ${object}`);
-}
-
-export function formatDate(date: Date | { toDate: () => Date }): string {
```

Comment:
I can't find anywhere externally that we are using this, so should be fine to remove.